django_celery_results/models.py

```diff
@@ -143,9 +144,71 @@ def group_result(self, app=None):
             app (Celery): app instance to create the GroupResult with.
 
         """
-        return GroupResult(
+        return CeleryGroupResult(
             self.group_id,
             [result_from_tuple(r, app=app)
              for r in json.loads(self.sub_tasks)],
-            app=app,
         )
+
+
+class GroupResult(models.Model):
+    """Task Group result/status."""
+
+    group_id = models.CharField(
+        max_length=getattr(
+            settings,
+            "DJANGO_CELERY_RESULTS_TASK_ID_MAX_LENGTH",
+            255
+        ),
+        unique=True,
+        db_index=True,
```

Comment:
instead of using db_index in the field isn't it better to use the model meta Index option and more future proof?

Reply:
Reading your comment more closely I've honestly never heard of the `model meta index option`.. I cant find anything in the django docs, you have a reference you could point me towards? 

I copied `db_index`  over from the "shared"  fields with the `TaskResult` model. If you're suggesting a change to that model as well would that be better as a separate PR?

Reply:
> instead of using db_index in the field isn't it better to use the model meta Index option and more future proof?

Not sure if it's needed. Defining the index in the meta part, is more for defining a different more exotic type of index (ie for postgres: https://docs.djangoproject.com/en/3.2/ref/contrib/postgres/indexes/) ,  or a partial index rather keeping an ol' trusty b-tree index.  https://docs.djangoproject.com/en/3.2/ref/models/options/#django.db.models.Options.indexes
 
Actually, you could be even less verbose by omitting the db_index=True altogether as 
> Note that when unique is True, you don’t need to specify db_index, because unique implies the creation of an index. [src](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.unique)

Reply:
interesting! I'm good with removing `db_index=True` if its redundant

Reply:
@auvipy thoughts?

Reply:
db_index is now implemented as model meta like this https://docs.djangoproject.com/en/3.2/ref/models/options/#django.db.models.Options.indexes instead of defining in the filed they can be defined in the model meta option